stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -30,6 +30,11 @@ julia> adjoint(A)
 2×2 adjoint(::Matrix{Complex{Int64}}) with eltype Complex{Int64}:
  3-2im  8-7im
  9-2im  4-6im
+
+julia> copy(adjoint(A)) # materialize the adjoint matrix
```

Comment:
Note that copy only does this on matrices, not on adjoint vectors:
```
julia> copy([1,2]')
1×2 adjoint(::Vector{Int64}) with eltype Int64:
 1  2

julia> copy([1 2; 3 4]')
2×2 Matrix{Int64}:
 1  3
 2  4
```

Reply:
Oh, interesting. Maybe it's best to not include this, then? The current docstring states "To materialize the view use `copy`", which may also not be fully accurate. Should we suggest `collect` instead, which seems to work in both these cases?

Reply:
It's a bit of a zoo. `copy` does copy, but returns another adjoint vector, I guess because these behave differently to 1-row matrices. 

With a GPU, `cu(ones(3))' |> collect` makes a Vector, while `cu(ones(3,3))' |> copy` makes a CuArray{T,2}.

Reply:
I've removed the examples using `copy` for now